src/styles/_catalog.scss

```diff
@@ -16,6 +16,12 @@
     margin-bottom: 48px;
   }
 
+  &__card:hover {
+    transform: scale(120%);
+    color: #00acdc;
```

Comment:
please change the color to the one asked in the task(it should be darker)

Reply:
yep, it shoud be #34568b